SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDevAuthTokenGenerator.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKDevAuthTokenGenerator.h"
+
+#include "Async/Async.h"
+#include "Framework/Notifications/NotificationManager.h"
+
+#include "SpatialCommandUtils.h"
+#include "SpatialGDKEditorSettings.h"
+#include "SpatialGDKSettings.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialGDKDevAuthTokenGenerator);
+
+FSpatialGDKDevAuthTokenGenerator::FSpatialGDKDevAuthTokenGenerator()
+	: bIsGenerating(false)
+{
+}
+
+void FSpatialGDKDevAuthTokenGenerator::DoUpdateSettings(FString DevAuthToken)
+{
+	AsyncTask(ENamedThreads::GameThread, [this, DevAuthToken]()
+		{
```

Comment:
Formatting nit: start these blocks on the same level as `AsyncTask`.
```suggestion
	{
```

Reply:
I prefer it too, and I generally don't understand why visual studio defaults to adding this additional level of indentation... I've generally begun spending less time fighting the visual studio defaults, just because it distracts me from writing the actual code to constantly be deleting extra tabs, I could change the defaults... but these days I usually just keep the VS defaults.